packages/editor/src/components/post-status/index.js

```diff
@@ -10,6 +10,10 @@ import {
 	TextControl,
 	RadioControl,
 } from '@wordpress/components';
+/**
+ * Internal dependencies
+ */
+import PostStickyPanel from '../post-sticky';
```

Comment:
```suggestion
import PostSticky from '../post-sticky';
```

This is no longer a "panel," so `PostSticky` may be easier to understand. Furthermore, the "Internal dependencies" comment is duplicated, so I recommend moving the import statement next to [here](https://github.com/WordPress/gutenberg/pull/62782/files#diff-a269f8ecddb6e11f49e3a7f19d800060dd3d76293c3ec8c8cc02e1e117975d11L29).